.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -1,60 +1,126 @@
+    validations:
+      required: true
 
-        Instead, I expect …
+  - type: textarea
+    id: repro-steps
+    attributes:
+      label: Reproduction steps
+      description: Provide clear, step-by-step instructions that anyone can follow to reproduce this issue.
+      placeholder: |
+        1. Open Swiftfin and navigate to your library
+        2. Select a video file with multiple audio tracks
+        3. Start playback
+        4. Attempt to access playback controls by tapping the screen
+        5. Observe that controls do not appear
+    validations:
+      required: true
+
+  - type: dropdown
+    id: platform
+    attributes:
+      label: Platform
+      description: Select the platform(s) where this bug occurs
+      multiple: true
+      options:
+        - iOS
+        - tvOS
     validations:
       required: true
+
   - type: input
     id: app-version
     attributes:
       label: Application version
       description: The version of the installed Swiftfin app.
-      placeholder: 1.0.0(58)
+      placeholder: 1.3.1, 1.0.1
     validations:
       required: true
+
   - type: dropdown
     id: installation-source
     attributes:
       label: Where did you install the app from?
       description: Choose the appropriate app store or installation method.
       options:
-        - TestFlight
         - App Store
+        - TestFlight
+        - GitHub
+
   - type: input
     id: device-info
     attributes:
       label: Device information
-      description: model
-      placeholder: iPhone 12 Pro, 12.9-in. iPad Pro (4th generations) 
+      description: Device model
+      placeholder: iPhone 18 Pro, 12.9-in. iPad Pro (6th gen), Apple TV 4K (2nd gen)
     validations:
       required: true
+
   - type: input
     id: os-version
     attributes:
       label: OS version
-      description: Version of the OS and other information
-      placeholder: iOS 14.7, iOS 15 beta 4 (19A5307g), iPadOS 14.7
+      description: Version of the operating system
+      placeholder: iOS 18.1, iPadOS 18.1, tvOS 18.1
     validations:
       required: true
+
   - type: input
     id: server-version
     attributes:
       label: Jellyfin server version
-      description: If on unstable, please specify the commit hash.
-      placeholder: 10.7.6
+      description: The version of your Jellyfin server. If on unstable, please specify the commit hash.
+      placeholder: 10.11.0
     validations:
-      required: true
\ No newline at end of file
+      required: true
+
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots or videos
+      description: Attach relevant screenshots or videos that demonstrate the issue.
+
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output if available. This can help diagnose the issue faster.
+      render: shell
+
+  - type: textarea
```

Comment:
I would prefer if all general information of the bug was just in the description, rather than having an "additional" section.